bin/convert-argv.js

```diff
@@ -15,6 +15,12 @@ module.exports = function(optimist, argv, convertOptions) {
 		process.exit(-1); // eslint-disable-line
 	}
 
+	// Version
+	if(argv.v || argv.version) {
+		console.log('v' + require("../package.json").version);
```

Comment:
Does `v` add any value? If not, I would drop it (easier to parse if someone wants to do that for some reason). I'm not so sure of the inline `require`. It might be better to push it to the beginning of the file.